web/controllers/v1/relay_group_membership_controller.ex

```diff
@@ -12,27 +12,36 @@ defmodule Cog.V1.RelayGroupMembershipController do
     render(conn, Cog.V1.RelayGroupView, "relays.json", relay_group: relay_group)
   end
 
+  def manage_membership(conn, %{"id" => id, "relays" => member_spec}),
+  do: manage_association(conn, :relay, %{"id" => id, "members" => member_spec})
+
+  def manage_assignment(conn, %{"id" => id, "bundles" => member_spec}),
+  do: manage_association(conn, :bundle, %{"id" => id, "members" => member_spec})
+
   # Manage membership of a relay group. Adds and deletes can be submitted and
   # processed in a single request.
-  def manage_membership(conn, %{"id" => id, "relays" => member_spec}) do
+  def manage_association(conn, type, %{"id" => id, "members" => member_spec}) do
     result = Repo.transaction(fn() ->
       group = Repo.get!(RelayGroup, id)
 
-      to_add = Enum.reduce(Map.get(member_spec, "add", []),
-                           %{found: [], not_found: []}, &lookup_or_fail/2)
-      to_remove = Enum.reduce(Map.get(member_spec, "remove", []),
-                           %{found: [], not_found: []}, &lookup_or_fail/2)
+      to_add = Enum.reduce(Map.get(member_spec, "add", []), %{found: [], not_found: []},
```

Comment:
fwiw I originally lifted most of this logic from `group_membership_controller.ex` for consistency and speed of implementation. 

https://github.com/operable/cog/blob/master/web/controllers/v1/group_membership_controller.ex#L21